src/main/java/com/sap/sgs/phosphor/fosstars/model/qa/AbstractTestVector.java

```diff
@@ -93,14 +108,16 @@ public boolean equals(Object o) {
       return false;
     }
     AbstractTestVector that = (AbstractTestVector) o;
-    return expectedNotApplicableScore == that.expectedNotApplicableScore
+    return expectedUnknownScore == that.expectedUnknownScore
```

Comment:
I am not sure, but would `Boolean.compare()` make more sense?

Reply:
`Boolean.compare()` returns an integer. If we used this method, the code would look like `Boolean.compare(expectedUnknownScore, that.expectedUnknownScore) == 0`. I don't think it's worth it.